src/cli/utils.js

```diff
@@ -307,50 +307,31 @@ export function matchKssDir(filepath, huron) {
 }
 
 /**
- * Merge JSON files for css modules classnames in a provided directory
+ * Ingest the classnames from the file specified in the huron config.
  *
- * @function mergeClassnameJSON
- * @param {string} directory - directory containing classname JSON files
+ * @function getClassnamesFromJSON
+ * @param {string} filepath - file containing classname JSON files
  *
- * @return {object} classnamesMerged - merged classnames. contents of each JSON file is nested within
+ * @return {object}           merged classnames. contents of each JSON file is nested within
```

Comment:
should probably update this description to just be `contents of classnames JSON file`